cmd/dashboard.go

```diff
@@ -56,6 +70,13 @@ var uploadDashboard = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 
 		filter := service.NewDashboardFilter(parseDashboardGlobalFlags(cmd)...)
+
+		confirm, _ := cmd.Flags().GetBool("skip-confirmation")
```

Comment:
Is this backwards? It looks like if I set --skip-confirmation that's when it will actually do confirmation.

Reply:
Yeah, fixed this and re-worded a bit.